api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/dynamodb/AbstractDynamoDbProperties.java

```diff
@@ -45,6 +45,12 @@
      */
     private long writeCapacity = 10;
 
+    /**
```

Comment:
Please describe in better details, what "billing mode" is, and what it actually does and what the default means.

Reply:
also, if the setting defines a set of fixed values, remove the final type as a String and instead define an enum.

Reply:
The `software.amazon.awssdk.services.dynamodb.model.BillingMode` enum is not available in the `cas-server-core-api-configuration-model` subproject.  I used [BaseAmazonWebServicesProperties.retryMode](https://github.com/apereo/cas/blob/9b7feef4f5d3132688a33af19855b2be878355c9/api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/aws/BaseAmazonWebServicesProperties.java#L109) as an example.  

The options are:
* Add the aws client lib to the dependencies of `cas-server-core-api-configuration-model` and make `AbstractDynamoDbProperties.billingMode` of type `software.amazon.awssdk.services.dynamodb.model.BillingMode`.
* Create a new enum that mirrors `software.amazon.awssdk.services.dynamodb.model.BillingMode` and make `AbstractDynamoDbProperties.billingMode` of that type
* Leave it as a `String`

What's your preference?

Reply:
The second option, almost always.

Reply:
Sounds good.